SP23.P03.Web/ClientApp/src/components/SignupModal.tsx

```diff
@@ -0,0 +1,104 @@
+import { Field, Form, Formik, FormikHelpers } from "formik";
+import React, { ReactElement, useEffect, useState } from "react";
+import { Button, Icon, Input, Modal, ModalProps } from "semantic-ui-react";
+import useSubscription, { notify } from "../hooks/useSubscription";
+import { CreateUserDto } from "../types/authentication";
+import { signupUser } from "./AuthProvider";
+
+export const openSignupModal = () => {
+    notify("open-signup-modal");
+}
+
+function Password(): ReactElement {
+    const [password, setPassword] = useState({
+        password: '',
+        confirmPassword: ''
+    })
+
+    const [match, setMatch] = useState(false);
+
+    const inputChange: (event: React.ChangeEvent<HTMLInputElement>) => void = (event) => {
+        const { value, name } = event.target;
+        setPassword({
+            ...password,
+            [name]: value
+        }) 
+    }
+
+    useEffect(() => {
+        setMatch(!!password.password && password.confirmPassword === password.confirmPassword)
```

Comment:
```suggestion
        setMatch(!!password.password && password.password === password.confirmPassword)
```
did you mean to do this?

Reply:
Ahh, no. That was code that I was trying to use to validate a user's password to ensure they matched; however, I couldn't get it working and was afraid that it wouldn't be done before the code freeze. I'll go ahead and remove it so that it's not extra code just sitting there.